rating/rating.css

```diff
@@ -0,0 +1,20 @@
+:root {
```

Comment:
you should  keep root styles in one file,

Reply:
Done